pkg/validation/limits.go

```diff
@@ -564,9 +564,10 @@ func (o *Overrides) MaxQueryLookback(_ context.Context, userID string) time.Dura
 	return time.Duration(o.getOverridesForUser(userID).MaxQueryLookback)
 }
 
+// Deprecated, can be removed in next major version
 // EvaluationDelay returns the rules evaluation delay for a given user.
 func (o *Overrides) EvaluationDelay(userID string) time.Duration {
-	return time.Duration(o.getOverridesForUser(userID).RulerEvaluationDelay)
+	return time.Duration(0)
```

Comment:
we should return this value even if it's deprecated right? should we add a warn message instead?